tests/robot-tests/tests/general_public/prod_table_tool.robot

```diff
@@ -0,0 +1,131 @@
+*** Settings ***
+Resource            ../libs/public-common.robot
+
+Suite Setup         user opens the browser
+Suite Teardown      user closes the browser
+
+Force Tags          GeneralPublic    Prod
+
+*** Test Cases ***
+Go to 'data-tables' page
+    user navigates to data tables page on public frontend
+
+Check page contains correct themes
+    user checks page contains element
```

Comment:
I'd create a new keyword in common for these cases in common.robot - `user checks page for details dropdown`. Should be able to use the same selector that `user opens details dropdown` uses - `xpath:.//details/summary[contains(., "${text}")]`

Reply:
You could also use this new keyword in other places in this test suite instead of just searching for the text itself.

Reply:
You've not created a new keyword for the checks in this test case.

I envision the keywords in this test case looking like  `user checks page for details dropdown    Children's social care`. See above comments.

Reply:
ah my bad, not sure how I missed this. I've created two new keywords: 
- `user expands detail section`: Responsible for opening theme headings 
- `user checks page for details dropdown`: Responsible for asserting whether a topic is visible